docs/guidelines/content-guidelines/01-content-strategy.md

```diff
@@ -117,6 +117,10 @@ Contains installation and configuration instructions, backup info, security docu
 
 Contains detailed information, such as architecture diagrams, configuration charts, etc.; note that the CLI command documents are generated automatically.
 
+### Modules
```

Comment:
I think it's weird that one node is called "Kyma Components" and the other one "Modules" without "Kyma". 
I'd argue we could change the navigation node to "Kyma Modules" and adjust it here too.